fuzzers/others/libafl-fuzz/src/scheduler.rs

```diff
@@ -127,14 +112,16 @@ where
     }
 }
 
-impl<S, Q, CS, F, M, O> HasQueueCycles for SupportedSchedulers<S, Q, CS, F, M, O>
+impl<CS, F, I, M, O, S, Q> HasQueueCycles for SupportedSchedulers<CS, F, I, M, O, S, Q>
```

Comment:
I still don't get how more generics is better, here?

Reply:
It's better because i get rid of UsesState and UsesInput.
And using `UsesState` and `UsesInput` was the *wrong* solution if you want to reduce the number of generics.

Reply:
Well having the input associated with a state doesn't sound necessarily wrong

Reply:
yeah so i'm not removing UsesInput from StdState

Reply:
what addison said is that you should only have associated X when you can define the getter to it

Reply:
but in this PR.
you can't implement fn state() or fn input() for this scheduler, then you don't implement UsesState, UsesInput.

Reply:
Not sure I fully understand, but just do what you thank and we see how it looks?